src/server/index.js

```diff
@@ -1,28 +1,92 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 const {
-  getSchema: storeApiGetSchema,
-  getContextFactory: storeApiGetContextFactory,
-} = require('@faststore/api')
+  envelop,
```

Comment:
Why does this file need to be a js one? Can't do something like this, can?

```ts
// src/server/index.ts
...
export {}
```

```js
// /src/server/index.js
const { } = require('./dist/server/index')
```

Reply:
it needs to be js because `gatsby-config.js` uses it